osquery/tables/system/windows/certificates.cpp

```diff
@@ -323,6 +551,29 @@ void enumerateCertStore(const HCERTSTORE& certStore,
 
   if (certContext == nullptr && GetLastError() == CRYPT_E_NOT_FOUND) {
     TLOG << "    Store was empty.";
+
+    // Personal stores for other users come back as empty, even if they are not.
+    if (storeName == "Personal" && !username.empty()) {
```

Comment:
You've got two layers of `if` statements here with no extra logic in between, can they be consolidated and remove a layer of scoping?

Reply:
I initially wrote it like that to avoid a crazy huge boolean condition, but they can definitely be consolidated into one. I'm wary of the condition growing hard to read though.. what do you think of this idea where we introduce some intermediate variables mostly for the sake of readability and being able to name the subconditions? 

```
    auto is_personal_store = storeName == "Personal" && !username.empty();
    auto not_already_added = storeLocation != "Users" || boost::ends_with(storeId, "_Classes");

    if (is_personal_store && not_already_added) {
```

Reply:
done 👍